### PR TITLE
Small fixes for qam-kgraft

### PR DESCRIPTION
**Pull request by mimi1vx**
1) wrong regepxp in reboot_restore.pm
2) wron use of tee command
3) wrong order of commands before 'snapshot after'
4) add BUILD variable to snapshot description

http://argus.suse.cz/tests/381

### CODE REVIEW

File: tests/qam-kgraft/reboot_restore.pm [hunk 1]
```diff
@@ -59,8 +59,8 @@ sub run() {
     script_run("cat /tmp/lsboot");
     save_screenshot;
 
-    script_run("basename /boot/initrd-\$(uname -r) | sed s_initrd-__g | sed s_-default__g | tee > /dev/$serialdev", 0);
-    my ($kver) = wait_serial(qr/^\d\.\d+\.\d+-\d+\.\d+/) =~ /(^\d\.\d+\.\d+-\d+\.\d+)\s/;
+    script_run("basename /boot/initrd-\$(uname -r) | sed s_initrd-__g | sed s_-default__g > /dev/$serialdev", 0);
```
Comment (okurz) [inline tests/qam-kgraft/reboot_restore.pm]: Why was tee wrong here?

Comment (mimi1vx) [inline tests/qam-kgraft/reboot_restore.pm]: ```tee > /dev/```... isn't correct construction

and regexp catch 3.12.14-1.1 but sometimes we have also ```uname -r``` = 3.12.14-1

Comment (okurz) [inline tests/qam-kgraft/reboot_restore.pm]: Yes, regex change makes sense.

Does `tee /dev/$serialdev` work?

Comment (mimi1vx) [inline tests/qam-kgraft/reboot_restore.pm]: here is important get result to serial console for use in next part of  script, not for print it to screen

Comment (okurz) [inline tests/qam-kgraft/reboot_restore.pm]: well I wonder why it has been there in the first place but nevermind :-)

**Pull request merged**